### PR TITLE
[yaml] Doc improvements

### PR DESCRIPTION
**Pull request by Polber**
**Explicitly define Beam imports for callables**
* PR adds explicit beam import statements to inlined python callables. This is technically redundant as 'apache_beam' already gets imported as 'beam' in several places along the codepath, so it is already imported when the callable is run. However, this is not guaranteed to stay this way, and it is more clear to the user than they must import dependencies which are not included on the hidden code path.

**Add suffix to windowing in a couple examples**
* A couple windowing examples did not include recently added time suffix

**Upgrade gcloud dataflow yaml command to use non-beta version**
* gcloud dataflow yaml run command was still referencing beta version

SQL Combine
* The example in the docs was not valid due to a small error in the transform normalizaiton step.
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Polber** [comment]
R: @robertwb

File: website/www/site/content/en/documentation/sdks/yaml-inline-python.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ define a `beam.PTransform` inline, e.g.
     constructor: __constructor__
     kwargs:
       source: |
+        import apache_beam as beam
```
Comment (robertwb) [inline website/www/site/content/en/documentation/sdks/yaml-inline-python.md:126]: Beam is explicitly and intentionally in the namespace for convenience: https://github.com/apache/beam/blob/release-2.41.0/sdks/python/apache_beam/utils/python_callable.py#L105 so imports are superfluous and shouldn't be added. (There aren't any other "hidden" imports we have to worry about.)

File: sdks/python/apache_beam/yaml/yaml_combine.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def normalize_combine(spec):
     def normalize_agg(dest, agg):
       if isinstance(agg, str):
         agg = {'fn': agg}
-      if 'value' not in agg and spec.get('language') != 'sql':
+      if 'value' not in agg and config.get('language') != 'sql':
```
Comment (robertwb) [inline sdks/python/apache_beam/yaml/yaml_combine.py:73]: Nice catch.

**Review by robertwb — COMMENTED** [review]
LGTM modulo the imports change.

File: website/www/site/content/en/documentation/sdks/yaml-inline-python.md (hunk 1 above)
Comment (Polber) [reply website/www/site/content/en/documentation/sdks/yaml-inline-python.md]: Ah I wasn't sure where the import was added so thanks for the link. In that case, I went the other direction and removed an import statement from one of the examples

**Review by robertwb — APPROVED** [review]

**Comment by damccorm** [comment]
@Polber @robertwb is this ready to merge? The test failure looks like a flake to me, though I'll rerun to see if we can get green

**Comment by Polber** [comment]
> @Polber @robertwb is this ready to merge? The test failure looks like a flake to me, though I'll rerun to see if we can get green

@damccorm Yes it is ready to merge

**Review by damccorm — APPROVED** [review]

**Pull request merged**